addons/stock/models/stock_move.py

```diff
@@ -1314,6 +1314,8 @@ def _action_confirm(self, merge=True, merge_into=False):
 
         return moves
 
+    def _should_explode_move(self, move):
+        return True
```

Comment:
A breakline is missing
```suggestion
        return True

```